src/Sql/dbo/Stored Procedures/Collection_ReadSharedCollectionsByOrganizationId.sql

```diff
@@ -0,0 +1,14 @@
+﻿CREATE PROCEDURE [dbo].[Collection_ReadSharedCollectionsByOrganizationId]
+    @OrganizationId UNIQUEIDENTIFIER
+AS
+BEGIN
+    SET NOCOUNT ON
+
+    SELECT
+        *
+    FROM
+        [dbo].[CollectionView]
+    WHERE
+        [OrganizationId] = @OrganizationId AND
+        [Type] != 1 -- Exclude DefaultUserCollection
```

Comment:
⛏️ for future proofing: this reads shared collections only, so checking for `[Type] = 0` is probably safer in case we add more collection types in the future.